src/Cell.cpp

```diff
@@ -764,6 +615,16 @@ void Cell::setName(const char* name) {
 
 
 /**
+ * @brief Sets the name of the Cell
+ * @param name the Cell name string
+ */
+void Cell::setRegion(Region* region) {
+  // FIXME: this is a memory leak
+  _region = region;
```

Comment:
Yeah, I'm really not sure how to deal with this, aside from specifying (and documenting) a rule that a `Region` can be assigned to one and only one `Cell`.